bolt/include/bolt/Core/BinaryContext.h

```diff
@@ -276,12 +276,24 @@ class BinaryContext {
   void deregisterSectionName(const BinarySection &Section);
 
 public:
+  enum class ICFLevel {
+    None,
+    Safe, // Safe ICF for all sections.
+    All,  // Aggressive ICF for code, but safe ICF for data, similar to MSVC's
```

Comment:
I guess "Aggressive" or "Unsafe" would be a better name, unless you took inspiration for this enum from somewhere else.

Reply:
This should be similar to LLD options. I'm confused by the comment about data and MSVC.